src/test/java/uk/nhs/hee/tis/trainee/forms/api/validation/FormRPartAValidatorTest.java

```diff
@@ -197,4 +200,29 @@ void shouldNotValidateSubmittedForm() {
     assertThat("Unexpected number of errors.", fieldErrors.size(), is(0));
     verifyNoInteractions(formRPartARepositoryMock);
   }
+
+  @ParameterizedTest
+  @NullSource
+  @ValueSource(strings = {""})
+  void shouldReturnFieldErrorWhenSubmittedFormWteMissingOrNull(String value) {
+    formRPartADto.setLifecycleState(LifecycleState.SUBMITTED);
+    formRPartADto.setWholeTimeEquivalent(value);
+    List<FieldError> fieldErrors = validator.checkSubmittedFormContent(formRPartADto);
+    assertThat("Should return an error", fieldErrors.size(), is(1));
+  }
+
+  @Test
+  void shouldNotReturnFieldErrorWhenSubmittedFormWteValid() {
+    formRPartADto.setLifecycleState(LifecycleState.SUBMITTED);
+    formRPartADto.setWholeTimeEquivalent("0.99");
+    List<FieldError> fieldErrors = validator.checkSubmittedFormContent(formRPartADto);
+    assertThat("Should not return an error", fieldErrors.size(), is(0));
+  }
+
+  @Test
+  void shouldNotReturnFieldErrorWhenDraftForm() {
+    formRPartADto.setWholeTimeEquivalent(null);
```

Comment:
I know the form is created with `DRAFT` by default, but we should set it explicitly here anyway for clarity and to avoid future failures if the setup is changed.